planning/obstacle_cruise_planner/src/node.cpp

```diff
@@ -621,14 +651,15 @@ std::vector<Obstacle> ObstacleCruisePlannerNode::convertToObstacles(
     }
 
     // 3. Check if rough lateral distance is smaller than the threshold
+    double lat_dist_from_obstacle_to_traj;
     const double min_lat_dist_to_traj_poly = [&]() {
-      const double lat_dist_from_obstacle_to_traj =
+      lat_dist_from_obstacle_to_traj =
         motion_utils::calcLateralOffset(traj_points, current_obstacle_pose.pose.position);
```

Comment:
```
const double lat_dist_from_obstacle_to_traj =
        motion_utils::calcLateralOffset(traj_points, current_obstacle_pose.pose.position);
```
can be moved to the outside of the lambda function.

Reply:
Done! Thanks for the review